integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -215,6 +242,9 @@ spec:
             - name: conf-volume
               mountPath: /opt/alluxio/conf
               readOnly: false
+            - name: logs-volume
+              mountPath: /opt/alluxio/logs
+              readOnly: false
```

Comment:
Could you make this a configurable value in `values.yaml`? `.Values.master.volumes.log.path` or something like that? Just so you can DRY-ify the hardcoded usage of `/opt/alluxio/logs` in this file (and others if applicable).

Reply:
updated to use `.Values.hub.volumes.log.path` since it's a hub-only volume